<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowI2PAddresses</class>
 <widget class="QDialog" name="ShowI2PAddresses">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>If you want to use a permanent I2P-address you have to set a 'mydestination' option in the configuration file:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Ready to save parameter (If you want to use this address save this text in the configuration file and keep it secret):</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="privText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">min-height: 150px;
max-height: 150px;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Address (you can publish it):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="pubText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">min-height: 125px;
max-height: 125px;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Short base32-address:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="b32Line">
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">min-height: 25px;
max-height: 25px;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="privButton">
       <property name="styleSheet">
        <string notr="true">min-width: 100px;
max-height: 100px;</string>
       </property>
       <property name="text">
        <string>Copy My Destination</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pubButton">
       <property name="styleSheet">
        <string notr="true">min-width: 100px;
max-height: 100px;</string>
       </property>
       <property name="text">
        <string>Copy Public Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b32Button">
       <property name="styleSheet">
        <string notr="true">min-width: 100px;
max-height: 100px;</string>
       </property>
       <property name="text">
        <string>Copy B32 Address</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>privText</tabstop>
  <tabstop>pubText</tabstop>
  <tabstop>b32Line</tabstop>
  <tabstop>privButton</tabstop>
  <tabstop>pubButton</tabstop>
  <tabstop>b32Button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
